test/samples/validation.response.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Can this please be moved into test/samples/claim or even a new folder test/samples/claim/validation